airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/Application.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.container_orchestrator;
+
+import com.google.common.annotations.VisibleForTesting;
+import io.airbyte.commons.logging.LoggingHelper;
+import io.airbyte.commons.logging.MdcScope;
+import io.airbyte.container_orchestrator.orchestrator.JobOrchestrator;
+import io.airbyte.workers.process.AsyncKubePodStatus;
+import io.micronaut.runtime.Micronaut;
+import jakarta.inject.Named;
+import jakarta.inject.Singleton;
+import java.lang.invoke.MethodHandles;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Entrypoint for the application responsible for launching containers and handling all message
+ * passing for replication, normalization, and dbt. Also, the current version relies on a heartbeat
+ * from a Temporal worker. This will also be removed in the future so this can run fully async.
+ * <p>
+ * This application retrieves most of its configuration from copied files from the calling Temporal
+ * worker.
+ * <p>
+ * This app uses default logging which is directly captured by the calling Temporal worker. In the
```

Comment:
For my understanding - what does this mean?

Reply:
This was [carried over](https://github.com/airbytehq/airbyte/blob/4c0ffb6c4036179e1e8961608273f4c191cde55e/airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/ContainerOrchestratorApp.java#L49-L59) from the pre-micronaut version.  As this behavior didn't change, I left these comments as is.

Reply:
Ok! I was just wondering in case this PR had logging changes.